packages/atomic/src/components/atomic-modal/atomic-modal.pcss

```diff
@@ -0,0 +1,88 @@
+@import '../../global/global.pcss';
+
+[part='backdrop'] {
+  pointer-events: none;
+
+  @screen desktop-only {
+    display: grid;
+    place-items: center;
+    grid-template-areas:
+      '. .     .'
+      '. modal .'
+      '. .     .';
+    grid-template-columns: 1fr auto 1fr;
+    grid-template-rows: 1fr auto 3fr;
+
+    transition: background-color 500ms ease-in-out;
+    &.opened {
+      background-color: rgba(40, 40, 40, 0.8);
+    }
+  }
+
+  &.opened {
+    pointer-events: auto;
+  }
+}
+
+atomic-focus-trap {
+  display: contents;
+}
+
+[part='container'] {
+  grid-area: modal;
+  overflow: hidden;
+
+  @screen mobile-only {
+    position: absolute;
+    left: 0;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    box-shadow: 0 2px 3px rgba(0, 0, 0, 0.1);
+  }
+
+  @screen desktop-only {
+    border-radius: var(--atomic-border-radius);
+    min-width: 30rem;
+    max-width: 100vw;
+    max-height: 100vh;
+  }
+}
+
+[part='header-wrapper'] {
+  @screen mobile-only {
+    padding: 1.5rem;
+  }
+
+  @screen desktop-only {
+    padding: 1rem 1.5rem;
+  }
+}
+
+[part='header'] {
+  @screen desktop-only {
+    font-weight: bold;
+  }
+}
+
+[part='body-wrapper'] {
+  @screen mobile-only {
+    padding: 2rem 1.5rem 1.25rem 1.5rem;
+  }
+
+  @screen desktop-only {
+    padding: 1.5rem;
+  }
+}
+
+[part='footer-wrapper'] {
+  @screen mobile-only {
+    @apply shadow-lg;
```

Comment:
Can we have more Tailwind classes like this rather than custom CSS? Could be done in another PR